src/io/flutter/FlutterUtils.java

```diff
@@ -75,6 +76,19 @@ public static VirtualFile getRealVirtualFile(@Nullable PsiFile psiFile) {
     return psiFile != null ? psiFile.getOriginalFile().getVirtualFile() : null;
   }
 
+
+  /**
+   * Returns the Dart file for the given PsiElement, or null if not a match.
+   */
+  public static @Nullable DartFile getDartFile(final @Nullable PsiElement elt) {
```

Comment:
Nit: Nullable annotation should be first token in signature, and parameter statement, like lines 57 and 73 above